www/package.json

```diff
@@ -74,7 +74,7 @@
     "yup": "^0.29.3"
   },
   "resolutions": {
-    "@babel/core": "7.11.6",
-    "@babel/parser": "7.11.5"
+    "@babel/core": "7.10.5",
+    "@babel/parser": "7.10.5"
```

Comment:
we should be able to use a ast-types resolution here instead of the babel ones. The underlying issue i think is ast-types version mismatch across react-docgen packages

Reply:
Cool confirmed this worked. Thanks